<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Tareas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Figuras</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Editor de texto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Inventario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>731</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Administrador de tareas</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>561</width>
        <height>401</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>10</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Crear tarea</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxDia">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>80</y>
          <width>72</width>
          <height>24</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>21</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>22</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>23</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>25</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>26</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>27</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>28</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>29</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="comboBoxMes">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>80</y>
          <width>111</width>
          <height>24</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Enero</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Febrero</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Marzo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Abril</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mayo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Junio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Julio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Agosto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Septiembre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Octubre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Noviembre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Diciembre</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>70</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Fecha:</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>120</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Diaria</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>150</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Semanal</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>200</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Descripcion:</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="descTarea">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>220</y>
          <width>321</width>
          <height>101</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>330</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Crear</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>40</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Titulo:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="tituloTarea">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>151</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="estatusTarea">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>370</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Tarea creada</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QListWidget" name="listaDeTareas">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>256</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Lista de tareas</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="descripcionDeTareas">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>60</y>
          <width>261</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>40</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Descripcion</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Crear tarea</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Ver tareas</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Figuras</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>150</y>
        <width>421</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>90</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Triangulo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rectangulo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Circulo</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Figura:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Editor de texto</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="paginaWord">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>401</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>160</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Negrita</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>200</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Cursiva</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>240</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Subrayado</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Gestion de inventario</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Modificar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>431</width>
        <height>401</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page_7">
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>20</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Agregar</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Precio</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>130</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Desc...:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>200</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Tipo</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>200</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Electronico</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ropa</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Alimento</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="pushButton_15">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>340</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Agregar</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="leDesc">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>130</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="leNombre">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>50</y>
          <width>113</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="adicional1">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>250</y>
          <width>113</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="adicional2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>290</y>
          <width>113</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="l2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>290</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Garantia</string>
        </property>
       </widget>
       <widget class="QLabel" name="l1">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>250</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Modelo</string>
        </property>
       </widget>
       <widget class="QLabel" name="statusAgr">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>370</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Se ha agregado</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>90</y>
          <width>111</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_8">
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>20</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Modificar</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_9">
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>20</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Eliminar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_16">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>280</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Eliminar</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listaEliminar">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>321</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="sttsElim">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>330</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Se ha eliminado</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
